package.json

```
@@ -20,12 +20,12 @@
   "homepage": "https://neo4j.com/developer/guide-neo4j-browser/",
   "main": "dist/index.html",
   "scripts": {
-    "build": "NODE_ENV=\"production\" webpack --config ./build_scripts/webpack.config.js --progress",
+    "build": "NODE_ENV=\"production\" webpack --config ./build_scripts/webpack.config.js",
     "dev": "jest --watch",
     "e2e": "cypress run",
     "e2e-local": "CYPRESS_E2E_TEST_ENV=\"local\" cypress run",
     "e2e-local-open": "CYPRESS_E2E_TEST_ENV=\"local\" cypress open",
-    "format": "prettier-eslint 'src/**/!(*.min).js' 'src/**/*.jsx' --write",
+    "format": "prettier-eslint --write 'src/**/!(*.min).{js,jsx,css,json}' 'src/**/*.{js,jsx,css,json}' 'e2e_tests/**/*.{js,jsx,css,json}' 'scripts/**/*.{js,jsx,css,json}' 'build_scripts/**/*.{js,jsx,css,json}'",
```

Seems like in `src` you're formatting all not-minified files, and then all files, could it be that you meant to do the following?
```suggestion
    "format": "prettier-eslint --write 'src/**/!(*.min).{js,jsx,css,json}' 'e2e_tests/**/*.{js,jsx,css,json}' 'scripts/**/*.{js,jsx,css,json}' 'build_scripts/**/*.{js,jsx,css,json}'",
```

---

^ good point

---

Thanks, fixed